README.md

```diff
@@ -857,6 +858,7 @@ releases:
   # *snip*
   hooks:
   - events: ["prepare", "cleanup"]
+    showlogs: true
```

Comment:
I prefer `logToStdout: true` which is a bit verbose but surfaces exactly what it does. WDYT?